src/main/resources/translator.properties

```diff
@@ -796,13 +799,14 @@ hudson.plugins.ws__cleanup.WsCleanup.patterns.type = OBJECT
 hudson.plugins.ws__cleanup.WsCleanup.patterns.hudson.plugins.ws__cleanup.Pattern = pattern
 hudson.plugins.ws__cleanup.WsCleanup.patterns.hudson.plugins.ws__cleanup.Pattern.type = OBJECT
 
+hudson.plugins.ws__cleanup.WsCleanup.patterns.hudson.plugins.ws__cleanup.Pattern.pattern = pattern
+
+hudson.plugins.ws__cleanup.WsCleanup.patterns.hudson.plugins.ws__cleanup.Pattern.type_child = type
+
 # If you have a child attribute called "type" refer to it as "type_child" in this file instead
 # See AbstractDSLStrategy.java for where that is handled.
 type = type
 type_child = type
-hudson.plugins.ws__cleanup.Pattern.pattern = pattern
-hudson.plugins.ws__cleanup.Pattern.type_child = type
```

Comment:
Is it safe to remove these? I thought they were used for another ws cleanup attribute

Reply:
I just added a more specific path here because that same pattern is within preBuildCleanup and it's handled differently.